frontend/packages/ux-editor/src/containers/DesignView/FormLayoutWarning.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import type { IInternalLayout } from '../../types/global';
+import { getDuplicatedIds } from '../../utils/formLayoutUtils';
+import { Paragraph } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import classes from './FormLayoutWarning.module.css';
+interface FormLayoutWarningProps {
```

Comment:
I'll suggest adding some space here 😄 
```suggestion

interface FormLayoutWarningProps {
```